pkgs/by-name/an/andcli/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "andcli";
+  version = "2.1.2";
+
+  subPackages = [ "cmd/andcli" ];
+
+  src = fetchFromGitHub {
+    owner = "tjblackheart";
+    repo = "andcli";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-xqLSKMdviwwSQY4D+h0NxjwSLgYLPqbmzZsg2gC95WM=";
+  };
+
+  vendorHash = "sha256-C5XW3nxTUjcH6YaFYSxuKdtMF5SvrbOjErWIQXNwSJA=";
+
+  meta = with lib; {
```

Comment:
(non-blocking) Could you move `with lib;` around a single expression for better readability?

Reply:
Yes, although is this not strictly enforced by any means